src/azurelinux/3.0/net10.0/build/Dockerfile

```diff
@@ -3,9 +3,11 @@ FROM mcr.microsoft.com/dotnet/nightly/runtime-deps:10.0-preview-azurelinux3.0
 
 RUN tdnf install -y \
         awk \
+        curl \
         nodejs \
         # provides 'useradd', required by Azure DevOps
         shadow-utils \
         # Provides 'su', required by Azure DevOps
+        tar \
```

Comment:
nit: move this up so the comment still refers to util-linux

Reply:
Good catch. Fixed.